debug/programs/debug.c

```diff
@@ -53,9 +53,12 @@ int main()
 
     volatile int i = 0;
     int j = 0;
-    char *fox = "The quick brown fox jumps of the lazy dog.";
+    static char fox[43];
     unsigned int checksum = 0;
 
+    /* fox needs to be writable, but the string could be in ROM. */
+    strcpy(fox, "The quick brown fox jumps of the lazy dog.");
+
```

Comment:
If you revert to the original code, but change `char *fox = "...";` to `char fox[] = "...";`, it'll accomplish what you want without the need for a static variable or strcpy.

Reply:
That's a subtlety I hadn't considered. Thanks.